src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+                        System.out.println("☹ OOPS!!! The description of a deadline cannot be empty.");
+                    }
+    
+                } else if (checkType[0].equals("event")) { // check event
+    
+                    String eventCondition = "/at "; 
+                    try {
+                        int indexOfTime = input.indexOf(eventCondition); //to find /
+                        String dateTime = input.substring(indexOfTime + eventCondition.length(), input.length()); // the date and time for by
+                        //convert to the correct one
+                        LocalDateTime eventTime = LocalDateTime.parse(dateTime, DateTimeFormatter.ofPattern("d/M/y Hmm"));
+                        String convertedTime = eventTime.format(DateTimeFormatter.ofPattern("MMM d yyyy hh:mm a"));
+                        String stringSliced = input.substring(6, indexOfTime); // after deadline
+                        stringArray.add(stringSliced);
+                        Event eventTask = new Event(stringSliced, convertedTime);
+                        taskArray.add(eventTask);
+                        String noOfTask = String.valueOf(taskArray.size());
+                        System.out.println(messageTask + eventTask.toString() + "\n"
+                                            + "Now you have " + noOfTask + " tasks in the list.");
+                    } catch (StringIndexOutOfBoundsException e) {
+                        System.out.println("☹ OOPS!!! The description of a event cannot be empty.");
+                    }
+                } else if (checkType[0].equals("delete")) {
+
+                    try {
+                        int index = Integer.parseInt(checkType[1]) - 1;
+                        String toBeRemoved = taskArray.get(index).toString();
+                        taskArray.remove(index);
+                        String noOfTask = String.valueOf(taskArray.size());
+                        String messageDeleted = "Noted. I've removed this task: \n" +
+                                                toBeRemoved + "\n" +
+                                                "Now you have " + noOfTask + " tasks in the list.";
+                        System.out.println(messageDeleted);
+
+                    } catch (ArrayIndexOutOfBoundsException e) {
+                        System.out.println("There are no tasks to be deleted!");
+                    }
+                } else {
+                    throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                }
+            } catch (DukeException e) {
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+            }
+            
+            input = sc.nextLine();
+        
+        }
+
+        // Recording everything in the duke.txt
+        for (int i = 0; i < taskArray.size(); i++ ) {
+            Task tasks = taskArray.get(i);
```

Comment:
Since it is referring to just 1 task, will it be better to name this variable as 'task' instead of 'tasks'?